website/src/app/page.tsx

```diff
@@ -147,7 +147,7 @@ export default function Page() {
         }
         image={
           <Image
-            src="/images/policy-conditions.png"
+            src="/images/policy-conditions.svg"
```

Comment:
Unfortunately this SVG is nearly 140KB which is double the compressed webp size that NextJS achieved:

<img width="709" alt="Screenshot 2024-08-21 at 2 12 59 PM" src="https://github.com/user-attachments/assets/b0190d30-a555-43b2-ba90-57bb5aebd0a9">

Reply:
I think the SVG contains lots of base64-encoded PNG data which is the reason why. Shadows gradients and such sometimes don't translate to SVG very well.

Reply:
I see, will reupload as PNG then